.circleci/config.yml

```diff
@@ -161,7 +161,7 @@ jobs:
       - deploy:
           name: Deploy laalaa to staging
           command: |
-            .circleci/deploy_to_kubernetes staging
+            .circleci/deploy_to_kubernetes database-migration-staging
```

Comment:
Have additional instruction to run database migration instead of overwriting staging/production deployments